sources/docker-compose.yml

```diff
@@ -141,7 +141,6 @@ services:
       - QUEUE_DASHBOARD_PASSWORD=${QUEUE_DASHBOARD_PASSWORD}
       - QUEUE_PREFIX=${QUEUE_PREFIX}
       - INSTITUTION_REQUEST_FOLDER=${INSTITUTION_REQUEST_FOLDER}
```

Comment:
can we please add the zonesftp env variables as part of your PR here in the queue-consumers? 
![image](https://user-images.githubusercontent.com/62901416/207393858-d14acf9a-d3c4-4739-9388-8188edc3cbc4.png)
its failing the make local command.